client-spec.md

```diff
@@ -262,7 +264,7 @@ The Verifier MUST then check the certificate against the verification policy. De
 
 ### 4.4. Transparency Log Entry
 
-By this point, the Verifier has already verified the signature by the Transparency Service ([§Establishing a Time for the Signature](#establishing-a-time-for-the-signature)). The Verifier MUST parse `body`: `body` is a base64-encoded JSON document with keys `apiVersion` and `kind`. The Verifier implementation contains a list of known [Transparency Service](https://docs.google.com/document/u/0/d/1NQUBSL9R64_vPxUEgVKGb0p81_7BVZ7PQuI078WFn-g/edit) formats (by `apiVersion` and `kind`); if no type is found, abort. The Verifier MUST parse `body` as the given type.
+By this point, the Verifier has already verified the signature by the Transparency Service ([§Establishing a Time for the Signature](#establishing-a-time-for-the-signature)). The Verifier MUST parse `body`: `body` is a base64-encoded JSON document with keys `apiVersion` and `kind`. The Verifier implementation contains a list of known Transparency Service [V1](./rekor-spec.md), [V2](./rekor-v2-spec.md)) formats (by `apiVersion` and `kind`); if no type is found, abort. The Verifier MUST parse `body` as the given type.
 
 Then, the Verifier MUST check the following; exactly how to do this will be specified by each type in [Spec: Sigstore Registries](./algorithm-registry.md):
 
```

Comment:
Do we have alternatives that include the different types you can upload to Rekor v2, either pre hashing the artifact vs a DSSE? Should we also provide guidance on supporting one or both?

Reply:
What do you mean by alternatives? 

but also, I guess I could make these links point to the specific type section in each rekor doc.

Reply:
I was referring to "4.6. Alternative Workflows". What I meant was, we be prescriptive about what the minimum set of requirements are for a client, and then the alternative section includes what's optionally supported. Do we require both hashedrekord and dsse, or just the former?